configuration_options.md

```diff
@@ -434,6 +394,3 @@ You can override or add configuration options in your `Gulpfile.js` by referenci
 elixir.config.production = true;
 ```
 
-# Custom Configuration File
```

Comment:
This was removed in the new version of Elixir.  We could work to put it back if we wanted.  Not too much work, but also not sure how much benefit.